README.md

```diff
@@ -1,27 +1,28 @@
-# Docs Build - Local Validation
+# Docs Validation
 
 [![Build Status](https://dev.azure.com/ceapex/Engineering/_apis/build/status/Docs%20Validation/docascode.vscode-docs-build?branchName=master)](https://dev.azure.com/ceapex/Engineering/_build/latest?definitionId=1716&branchName=master)
 
-This is a VS Code extension to provide the Local Build Validation experience for a docs repository.
+This extension enables you to run build validation on a Docs conceptual or Learn repo at author time in VS Code. This means you can make sure your repo is free of validation issues before making a pull request.
 
-## Features
+## Prerequisites
 
-- Ability to sign in/out of the dependant Docs APIs, and display the status in the status bar
-- Validate the current workspace folder
-- Display the real-time build streaming output in the `Docs` channel
-- Display all diagnostics in the `Problem` view with detailed information after a build has finished.
-![OAuth](https://github.com/docascode/vscode-docs-build/blob/master/resources/vscode-docs-build.gif?raw=true)
+- You must be a Microsoft employee.
+- Install [git](https://git-scm.com/downloads).
+- Clone your Docs or Learn repo locally in VS Code.
+- All files in the repo must be saved.
+
+## How to use the Docs Validation extension
 
-## Requirements
+1. Open a Docs conceptual or Learn repo in VS Code with the extension installed.
+1. Sign in by clicking **Docs Validation: Sign in to Docs** from the VS Code status bar.
+1. Trigger a build by clicking **Yes** when prompted after sign in, by clicking **Docs Validation** on the status bar then clicking **Validate** from the drop-down menu, or by right-clicking any file in the repo and selecting **Validate this workspace folder**.
+1. Build validation will run locally and all results will be output to the Problems pane.
 
-- [git](https://git-scm.com/downloads) installed
+![OAuth](https://github.com/docascode/vscode-docs-build/blob/master/resources/vscode-docs-build.gif?raw=true)
 
-## Performance benchmark
+> **Note:** The first time you validate a repo, all the Docs build dependencies will be fetched and cached locally. Subsequent validation runs will be faster.
 
-> Notes:
-> 1. On the first build of a Docs repository on your local machine, all the dependencies (template repository, Cross Repository Reference, build dependencies, etc.) will need to be fetched, and this will take some time to complete (depending on your network).
-> 2. All of the fetched resources will be cached locally, and additional local builds will run much faster.
-> 3. A restore will be performed on each VS Code session (VS Code).
+<!-- Can we pull these details from the public extension readme? The troubleshooting section might be helpful, but using images that was isn't accessible, so maybe we comment them out for now and build a better accessible readme when we add to the Docs Authoring Pack?
```

Comment:
what do you mean `using images that was isn't accessible`?

when we are saying `add the docs validation to Docs authoring pack`, we will just update the Docs authoring pack to reference this extension, so when the user installs (or installed) the Docs authoring pack, the Docs validation will also be installed, but we will keep this repository and publish the Docs validation extension from this repository. Just like other extensions in the pack like Docs markdown, they are also individual extensions which has been referenced by Docs authoring pack, the only difference is we use the separated repository for Docs validation.
So if we want to build a better accessible readme, we should update this one.

Reply:
The troubleshooting guide used images to show potential error messages. Users of screen readers would have no idea what they said. I couldn't really read them either. If we're going to ship this kind of information, we should add the error text, not screen shots.

Reply:
OK, then you can comment this part for now, and I will replace the screeeen shots with error text in another PR. So we can still keep this part in this readme file, right?

Reply:
Uncommented the Troubleshooting section and made some edits. Please replace the images with code blocks (```) showing the actual error text.